pkg/rbac/userData.go

```diff
@@ -101,7 +114,11 @@ func userCacheValid(user *UserDataCache) bool {
 // Equivalent to: oc auth can-i list <resource> --as=<user>
 func (user *UserDataCache) getClusterScopedResources(cache *Cache, ctx context.Context,
 	clientToken string) (*UserDataCache, error) {
-
+	var uid string
+	var err error
+	if uid, err = cache.GetUserUID(ctx); err != nil {
+		klog.Warning("Error getting user's uid: ", err)
```

Comment:
Are we able to proceed without the `uid` or should we return an error here?

Reply:
This uid is being used for logging only. Moved it.